backends/gdx-backend-robovm-metalangle/src/com/badlogic/gdx/backends/iosrobovm/bindings/metalangle/MGLKViewControllerDelegate.java

```diff
@@ -27,7 +27,7 @@
 /*</javadoc>*/
 /*<annotations>*//*</annotations>*/
 /*<visibility>*/public/* </visibility> */ interface /* <name> */ MGLKViewControllerDelegate/* </name> */
-	/* <implements> */extends NSObjectProtocol/* </implements> */ {
+	/* <implements> */ extends NSObjectProtocol/* </implements> */ {
```

Comment:
Are all these other formatting changes correct?

Reply:
I didn't add them, spotlessApply did. And that does look like a formatting violation of some kind.

Reply:
The Spotless formatter doesn't seem to care about these. If I run Spotless in master no changes are made but I've tested applying one of these changes manually and Spotless doesn't detect a format violation either. I'm not sure why Spotless complains in your environment, any chance you ran a different version of Spotless or formatter at some point in this branch?

Reply:
I had tried (briefly) the latest version of Spotless to see if it would resolve various warning messages Gradle spits out at the start of each build. They look like:
```
Configuration 'spotless-1429330901' was resolved during configuration time.
This is a build performance and scalability issue.
See https://github.com/gradle/gradle/issues/2298
Run with --info for a stacktrace.
Configuration 'spotless865455342' was resolved during configuration time.
This is a build performance and scalability issue.
See https://github.com/gradle/gradle/issues/2298
Run with --info for a stacktrace.
Configuration 'spotless-641085672' was resolved during configuration time.
This is a build performance and scalability issue.
See https://github.com/gradle/gradle/issues/2298
Run with --info for a stacktrace.
```
The latest version didn't affect the warnings, and there wasn't any way I could check the dozens of files spotless always claims to have modified (but didn't, it was just line ending differences every time -- except apparently there are a few new things Spotless checks in the latest version).